ibis/util.py

```diff
@@ -73,6 +74,36 @@ def is_one_of(values: Sequence[T], t: Type[U]) -> Iterator[bool]:
 all_of = toolz.compose(all, is_one_of)
 
 
+def coerce_to_dataframe(data: Any, names: List[str]) -> pd.DataFrame:
+    """Coerce the following shapes to a DataFrame.
+
+    The following shapes are allowed:
+    (1) A list/tuple of Series
+    (2) A Series of list/tuple
+
+    Parameters
+    ----------
+    val : pd.DataFrame, a tuple/list of pd.Series or a pd.Series of tuple/list
+
+    Returns
+    -------
+    pd.DataFrame
+    """
+    if isinstance(data, pd.DataFrame):
+        result = data
```

Comment:
yeah this is not a great function because it sometimes returns the same object and sometimes a new one.

Reply:
Out of curiosity, why is it not good to return same object sometimes and new one some other time?